### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Salamandar**
Manifestv2

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by Salamandar** [comment]
!bookwormtestme

**Review by alexAubin — APPROVED** [review]
Quick review, LGTM

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        autoupdate.strategy = "latest_github_release"
+        autoupdate.asset.arm64 = "syncthing-linux-arm64-*.tar.gz"
+        autoupdate.asset.amd64 = "syncthing-linux-amd64-*.tar.gz"
+        autoupdate.asset.i386 = "syncthing-linux-386-*.tar.gz"
+        autoupdate.asset.armhf = "syncthing-linux-arm-*.tar.gz"
+
+    [resources.ports]
+    main.default = 22000
+    main.exposed = "TCP"
```
[the remark this answers is not in the document]
Comment (Salamandar) [reply manifest.toml:62]: No, this is already handled by NAT mapping (uPNP). See for example on my laptop, Syncthing log:

> 2023-11-12 12:05:01 New NAT port mapping: external TCP address XXX.XXX.XXX.XXX:39729 to local address [::]:22000.

I can see on my router's UI the automatic bindings :

Capture d’écran du 2023-11-12 12-07-33

**Pull request merged**